src/normalizer.ts

```diff
@@ -404,14 +404,15 @@ class V1SchemaNormalizer {
       case 'string':
         return s.format === 'date-time' ? new DateTimeType(s) : new StringType(s)
       case 'integer':
-        return new Int32Type(s)
+        if (s.format === 'int32') return new Int32Type(s)
+        if (s.format === 'int64') return new Int64Type(s)
+        this.recordError(`IDK how to parse this integer: ${s.format}`);
```

Comment:
We should decide what to do when there is no format for type = number and integer. OpenAPI supports generic numbers and ints: https://swagger.io/docs/specification/v3_0/data-models/data-types/

I think perhaps we want to default them to double and int64 respectively. Maybe after we rollout int64 support everywhere.

Reply:
I'm gonna default to int64 as not to break any documents that may have been using integer without a format. so will need to update all the bindgens before upgrading this library.